lib/elixir/test/elixir/kernel/warning_test.exs

```diff
@@ -981,7 +981,7 @@ defmodule Kernel.WarningTest do
 
   # TODO: Simplify when we require Erlang/OTP 24
   if System.otp_release() >= "24" do
-    @argument_error_message "the call to :erlang.atom_to_binary/2"
+    @argument_error_message "the call to Atom.to_string/1"
```

Comment:
Not sure what prompted this change, is inlining working differently after the rewrite change?

Reply:
I'm also curious about this. Wondering if the change in elixir_rewrite.erl didn't work 🤔

Reply:
Maybe it didn't work because arguments stay the same, and in that case the change in `elixir_rewrite.erl` should be changed from using a "rewrite" macro to "inline" macro. Just guessing though.

Reply:
@michallepicki correct. It can now be an inline rule.

Reply:
I did that, but it still doesn't work as I would expect. Am I missing something?